app/src/main/java/org/listenbrainz/android/presentation/features/dashboard/DashboardActivity.kt

```diff
@@ -27,7 +30,7 @@ import org.listenbrainz.android.presentation.theme.ListenBrainzTheme
 
 @AndroidEntryPoint
 class DashboardActivity : ComponentActivity() {
-
+    @RequiresApi(Build.VERSION_CODES.TIRAMISU)
```

Comment:
Do we want to do this?

Reply:
"android.Manifest.permission.READ_MEDIA_AUDIO" permission in line 45 shows lint warning if we remove this.
Although this being here doesn't interrupt with my android 10 device so I think it should be fine

Reply:
Thanks for the clarification.